### DISCUSSION TITLE
July 22 Quansight Show & Tell

### DISCUSSION THREAD

**Discussion by isabela-pf** [Quansight/Quansight.github.io#35] [Show and tell]
July 22 show & tell notes. Please use this discussion to mark highlights and links from our discussions.

DO NOT DISCLOSE CONFIDENTIAL CUSTOMER INFORMATION

**Comment by honno** [comment]
Kush's new podcast https://rss.com/podcasts/whatsupkush/

**Reply by honno** [reply]
https://github.com/tonyfast/mee#readme https://tonyfast.github.io/tonyfast/tonyfast/readme.html

**Reply by honno** [reply]
https://www.gnu.org/software/emacs/manual/html_node/emacs/Keyboard-Macros.html

**Reply by honno** [reply]
wood working yt rec https://www.youtube.com/user/stevinmarin

**Comment by isabela-pf** [comment]
- Tony/creating a [readme](https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme ) profile on github: ~13 minutes–17 minutes
- Isabela/what does intersectionality mean?: ~17 minutes–31 minutes
- Aaron/emacs keyboard macros: ~32 minutes–42 minutes
- Dharhas/woodworking: ~44 minutes–53 minutes
- Dillon/[blenderGIS](https://github.com/domlysz/BlenderGIS): ~53 minutes–

**Reply by melissawm** [reply]
https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

**Comment by isabela-pf** [comment]
Thank you all for taking notes!
